transcode/transcode.go

```diff
@@ -92,7 +92,6 @@ func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName st
 	}
 
 	// Iterate through the segment URLs and transcode them
-	// TODO: Some level of parallelisation once we're happy this works well
```

Comment:
This is now being tracked under https://github.com/livepeer/catalyst-api/issues/111 so we can ditch the TODO